docs/guis/betabeat/settings.md

```diff
@@ -1,7 +1,499 @@
+  This field should be automatically set correctly, when choosing an accelerator in the [Beam Selection Window](beam_selection.md).
+
+- **Phase Advance via Union of BPMs**:
+  Uses the `how=outer` pandas method to join dataframes of multiple measurements, which is equivalent to a union of BPMs (i.e. uses all BPMs present in any measurement file),
+  instead of the default `how=inner` which creates an intersection of BPMs (i.e. uses only BPMs present in all measurement files).
+  Default: `False`.
+
+- **Nonlinear Calculations**:
+  Activate which "nonlinear" calculations should be performed: The calculation of Resonance Driving Terms (RDTs) and/or the calculation of combined RDTs.
+
+- **RDT Magnet Order**:
+  When RDT calculation is active, this field specifies the highest order of magnetic fields for which the corresponding RDTs are calculated.
+  Needs to be between 2 and 8 (including).
+  Default: `4` (octupoles).
+
+- **Use Three-Bpm Method**:
+  When active, the [Three-BPM Method][three_bpm_method] is used to calculate beta-from-phase, otherwise the [N-BPM Method][n_bpm_method] is used.
+  Default: `False`.
+
+- **Calculate only coupling**:
+  When active, skips most of the optics calcualtions and calculates only phases and coupling RDTs.
+  Useful for quick analysis or machines that do not support (e.g. not enough BPMs) some of the other analysis methods.
+  Default: `False`.
+
+- **3D-Excitation**:
+  Activate this, when [3D-Excitation/3D-Kicks][three_d_kicks] were used to calculate chromatic parameters directly from the longitudinal information of the harmonic analysis.
+  Default: `False`
+
+- **Isolation Forest Active**:
+  Activate for additional filtering of BPMs before analysis via the [Isolation Forest][isolation_forest_cleaning] machine learning method.
+  Default: `False`.<br>
+  :fontawesome-solid-triangle-exclamation:{.warning-colored} _[This method is currently under re-evaluation][isolation_forest_issue]._
+
+- **Calculate Second-Order Dispersion**:
+  Activate to calculate also second order dispersion.
+  Default: `False`.
+
+- **Calculate Chromatic Beating**:
+  Activate to calculate chromatic beatings: W, PHI and coupling.
+  Default: `False`.
+
+- **Coupling Method**:
+  Switch between the different coupling methods. Default: `2`.
+    - `0`: Disabled.
+    - `1`: 1 BPM Method.
+    - `2`: 2 BPMs Method.<br>
+
+- **Range of BPMs**
+  Select the range of BPMs for beta-from-phase calculation.
+  Can be an odd number between `5` and `15` BPMs (including).
+  Default: `11`.
+
+- **Compensation Mode**
+  Mode of phase compensation for the analysis in case of a driven beam excitation,
+  to remove the effect of the AC-Dipole (present in the measurements) from the actual phase advances in the machine (i.e. when the AC-Dipole is not used during the rest of the operation).
+  This can be `model`, `equation` or `none`.
+  Default: `model`.
```

Comment:
I hate that we have kept these names. I think we should provide some more info here at least:

- `none` means that we get raw measurement data.
- `model` applies a flat rescaling factor that is calculated from the tunes and tune deltas iirc.
- `equation` refers to the fully analytical compensation from R. Miyamoto (DOI:10.2172/1013520)

Reply:
Same here. Feel free to add, but I am actually not really knowledgable in this regard. AFAIK there are multiple different analytical ways one of them is implemented, but it has been mentioned that it should be reviewed.